pkg/core/mempool/mem_pool.go

```diff
@@ -49,6 +50,9 @@ type Pool struct {
 	inputs       []*transaction.Input
 	claims       []*transaction.Input
 
+	resendThreshold uint32
+	staleTxs        []*transaction.Transaction
```

Comment:
And you won't gonna need `staleTxs` here at all.

Reply:
Fixed.